backend/src/main/kotlin/org/loculus/backend/controller/SubmissionController.kt

```diff
@@ -347,6 +348,7 @@ open class SubmissionController(
         ) @RequestParam(required = false) statusesFilter: List<Status>?,
         @HiddenParam authenticatedUser: AuthenticatedUser,
         @RequestParam(required = false, defaultValue = "INCLUDE_WARNINGS") warningsFilter: WarningsFilter,
+        @RequestParam(required = false, defaultValue = "INCLUDE_ERRORS") errorsFilter: ErrorsFilter,
```

Comment:
So I think given the UI we will want to filter in the same way as we do the counting - including or not "PERFECT" or "WARNINGS" or "ERRORS"

Reply:
The API has now changed to include a processingResultFilter where you can include (or not) these states